product_docs/docs/biganimal/release/getting_started/identity_provider/auth0.mdx

```diff
@@ -0,0 +1,59 @@
+navTitle: "Auth0"
+description: "Using Auth0 as your identity provider for your Azure or AWS account"
+---
+
+
+A unique URL and access code is provided in an email from cloudcare@enterprisedb.com. Contact [cloudcare@enterprisedb.com](mailto:cloudcare@enterprisedb.com) if you don't receive the email. The URL becomes invalid after you set up your identity provider with BigAnimal. If you have issues with the code or identity provider setup, contact [Support](/biganimal/release/overview/support).
+
+## Set up BigAnimal with Auth0
+
+1. Open the link in the email sent from cloudcare@enterprisedb.com to access the Set Up Identity Provider page in BigAnimal.
+
+1. You will be copying and pasting between BigAnimal and Auth0 so in a separate browser tab or window, log into [auth0.com](https://auth0.com/) and navigate to the Applications page and select **Create Application**.
+
+1. In the Create application dialog, enter a name and choose **Native** as the application type. Select **Create**.
+1. On the Connections tab or the page for your application, enable the toggles for the databases containing your user names (Username-Password-Authentication, GitHub, or Google/Gmail).
+1. Select the Settings tab of the page for your application, scroll down, and expand the **Advanced Settings** section.
+1. Select the Certificates tab in Advanced Settings and download the certificate in PEM format.
+1. Select the Addon tab in Advanced Settings and enable the **SAML2 Web App** toggle. 
+1. Select the Settings tab on the Addon: SAML2 Web App dialog.
+1. Copy the Assertion Consumer Service URL value from BigAnimal and paste it as the Application Callback URL value in Auth0.
+1. Copy the Audience URI from the Set Up Identity Provide page in BigAnimal and paste it as the audience value in the Settings field. The following attributions statements may need modification depending on what form of your user names (for example, if the user names come from GitHub, there are no last name and first name fields):
+
+    - &lt;assertion_path>/name           
+    - &lt;assertion_path>/emailaddress   
+    - &lt;assertion_path>/nameidentifier 
+    - &lt;assertion_path>/surname                                                                               
+    - &lt;assertion_path>/givenname                                                                      
+   
+      Where &lt;assertion_path> is http://schemas.xmlsoap.org/ws/2005/05/identity/claims.
+ 
+   Add the following lines after the attribution statements:
+   ```
+   "createUpnClaim": false,
+   "passthroughClaimsWithNoMapping": false,
+   "mapUnknownClaimsAsIs": false,
+   "mapIdentities": false,
+   "nameIdentifierFormat": "urn:oasis:names:tc:SAML:2.0:nameid-format:email",
+   "nameIdentifierProbes": [
+   "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/emailaddress"
+   ]
+   ```
+   
+1. Select **Enable**.
+1. Select the Usage tab and copy Identity Provider Login URL.
```

Comment:
```If you have a custom domain, use the custom-domain-based URL rather than your Auth0 domain. ```

this is from https://auth0.com/docs/authenticate/protocols/saml/saml-sso-integrations/configure-auth0-saml-identity-provider